swift/apple/FirezoneKit/Sources/FirezoneKit/Features/SettingsView.swift

```diff
@@ -67,133 +105,341 @@ public struct SettingsView: View {
   @ObservedObject var model: SettingsViewModel
   @Environment(\.dismiss) var dismiss
 
-  let teamIdAllowedCharacterSet: CharacterSet
   @State private var isExportingLogs = false
 
   #if os(iOS)
     @State private var logTempZipFileURL: URL?
     @State private var isPresentingExportLogShareSheet = false
   #endif
 
+  struct PlaceholderText {
+    static let accountId = "account-id"
+    static let authBaseURL = "Admin portal base URL"
+    static let apiURL = "Control plane WebSocket URL"
+    static let logFilter = "RUST_LOG-style filter string"
+  }
+
+  struct FootnoteText {
+    static let forAccount = "Your account ID is provided by your admin"
+    static let forAdvanced = """
+      Use the default values unless you know what you're doing. \
+      Changing them might disrupt access to your Firezone resources.
```

Comment:
Perhaps we can make this text more prominent and also clarify that changing these fields is **not** supported? We're bound to have cowboys customize these, break and then open GitHub issues as a result 🤠 🙃 😃 

```suggestion
     WARNING: These are intended for debug purposes **only**. Changing these is unsupported and disrupt access to your Firezone resources.
```

Reply:
Fixed in commit [d4e9555](https://github.com/firezone/firezone/pull/2540/commits/d4e955569d360d7dd5eb50d3e67513b222a88e62).